insights/specs/__init__.py

```diff
@@ -215,6 +215,7 @@ class Specs(SpecSet):
     lspci_kernel = RegistryPoint()
     lssap = RegistryPoint()
     lsscsi = RegistryPoint()
+    ls_lib_fw = RegistryPoint()
```

Comment:
@vishwanathjadhav sorry about not asking this before you pushed your last commit, but wouldn't it be clearer to name the spec `ls_lib_firmware` to be consistent with the parser name and with the `ls_sys_firmware` spec?

Reply:
Done